routing_model/index.html

```diff
@@ -531,6 +531,55 @@ <h2>Process for Adult/Elder promotion and demotion including merge</h2>
      We have to be careful that the section follows up on relocation once merged, so we may want to avoid active relocation when merging.
   </p>
 
+  <button class="collapsible">CheckElderTimeout</button>
+  <div class="content">
+    <p>As for incrementing work units, we want to update the eldership status of all nodes in a section on a syncrhonised, regular basis.<br/>
+    For this reason, it makes sense to have a timer going through Parsec.<br/>
+    Note that this timer has to be relatively fast so that it remains highly unlikely that 1/3 of the elders in any section would go offline within one timer's duration.
+    </p>
+  </div>
+  <button class="collapsible">Merge RPC</button>
+  <div class="content">
+    <p>The merge RPC is the RPC a section sends to their neighbour section when they need merging.<br/>
+    In this flow, we handle both situations:<br>
+    <ul>
+      <li> Our neighbour triggers the merge and we receive their Merge RPC.<br>
+           We then vote for ParsecNeighbourMerge.
+      <li> We trigger the merge ourselves (see ProcessMerge flow)
+    </ul>
+    </p>
+  </div>
+  <button class="collapsible">ParsecNeighbourMerge</button>
+  <div class="content">
+    <p>We vote for this Parsec event on receiving a Merge RPC from our neighbour section.<br>
+    It contains the information about them that we need for merging. When this PARSEC event reaches consensus in PARSEC, we store that information by calling store_merge_infos.
+    </p>
+  </div>
+  <button class="collapsible">store_merge_infos</button>
+  <div class="content">
+    <p>This function is used to store the merge information from a neighbour section locally.<br>
+    Once it has been stored, has_merge_infos will return true and we will be ready to enter the ProcessMerge flow.
+    </p>
+  </div>
+  <button class="collapsible">has_merge_infos</button>
+  <div class="content">
+    <p>This function indicates that we received sufficient information from our neighbour section needing a merge, and reached consensus on it. We are ready to merge with them.
```

Comment:
> We are ready to merge with them.

Not necessarily with them at first in case of multi merge.
Also we are not yet ready to merge until we consensused OurMerge.